docs(component-store): update docs to add dictionnary of obs inside select (#3730)
ngrx/platform: projects/ngrx.io/content/guide/component-store/read.md

}
</code-example>

## Creating a View Model

Creating view models is a recommended way to consolidate multiple streams in a clean API to provide to your component template.

> The `select` method accepts a dictionnary of observables as input and return an observable of dictionnary as output. View models can be written the following way: 

Reviewer: ```suggestion
The `select` method accepts a dictionary of observables as input and returns an observable of the dictionary of values as output. View models can be written the following way: 
```

> The `select` method accepts a dictionary of observables as input and returns an observable of the dictionary of values as output. View models can be written the following way: 

Reviewer 2: ```suggestion
The `select` method accepts a dictionary of observables as input and returns an observable of the dictionary of values as output. View models can be written in the following way: 
```

<code-example header="movies.store.ts">

>   private readonly vm$ = this.select({
>     movies: movies$,
>     userPreferredMovieIds: userPreferredMovieIds$,
>     userPreferredMovies: userPreferredMovies$
>   });

Reviewer: ```suggestion
  private readonly vm$ = this.select({
    movies: this.movies$,
    userPreferredMovieIds: this.userPreferredMovieIds$,
    userPreferredMovies: this.userPreferredMovies$
  });
```

Author: well seen

  readonly currentPageIndex$ = this.select(state => state.currentPageIndex);

>   private readonly fetchMoviesData$ = this.select({
>     moviesPerPage: moviesPerPage$,
>     currentPageIndex: currentPageIndex$
>   },{debounce: true}, // 👈 setting this selector to debounce

Reviewer: ```suggestion
  private readonly fetchMoviesData$ = this.select({
    moviesPerPage: this.moviesPerPage$,
    currentPageIndex: this.currentPageIndex$
  },{debounce: true}, // 👈 setting this selector to debounce
```
